server/cassandra/seedCassandra.js

```diff
@@ -0,0 +1,55 @@
+const cassandra = require('cassandra-driver');

```

Comment:
Is this file a duplicate of your queryCassandra.js file?  They both seem to be attempting to seed the database.

Reply:
It almost was, it's a little messy, I need to delete some things from that file, thanks for pointing that out.